src/Data/List/Relation/Binary/Permutation/Inductive/Properties.agda

```diff
@@ -233,6 +233,11 @@ module _ {a} {A : Set a} where
     { isCommutativeMonoid = ++-isCommutativeMonoid
     }
 
+  -- Another useful lemma
+
```

Comment:
This seems like the plural of `shift` (instead of moving one value, you're moving
a whole list of them). So I'd suggest the name `shifts`.